install_config/install/advanced_install.adoc

```diff
@@ -1678,22 +1675,145 @@ specifications, and save it as *_/etc/ansible/hosts_*.
 == Running the Advanced Installation
 
 After you have xref:configuring-ansible[configured Ansible] by defining an
-inventory file in *_/etc/ansible/hosts_*, you can run the advanced installation
-using the following playbook:
+inventory file in *_/etc/ansible/hosts_*, you run the advanced installation
+playbook via Ansible. {product-title} installations are currently supported
```

Comment:
IBM Style Guide says it's OK when it means "by way of" referring to routing data. I try to use sparingly; this instance could probably just be "using".